### PR TITLE
feat: Make MinIO storage provider region setting configurable

### PR DESCRIPTION
**Pull request by hunterlord**
Fix: https://github.com/casdoor/casdoor/issues/3432

### CODE REVIEW

**Comment by hsluoyz** [comment]
@hunterlord will this PR affect the default behavior?

**Comment by hsluoyz** [comment]
@dacongda plz review

**Comment by hunterlord** [comment]
@hsluoyz 
I have already tested it locally, and it works fine. 😊

1. The region_id field displays correctly.
image
2. Updating and saving work as expected
image

3. MinIO connects successfully.
image

if update  region_id
image
will return 
image
minio region setting is:
image

File: storage/storage.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ func GetStorageProvider(providerType string, clientId string, clientSecret strin
 	case "AWS S3":
 		return NewAwsS3StorageProvider(clientId, clientSecret, region, bucket, endpoint), nil
 	case "MinIO":
-		return NewMinIOS3StorageProvider(clientId, clientSecret, "_", bucket, endpoint), nil
```
Comment (hsluoyz) [inline storage/storage.go:26]: The default value "_" should not be changed. You changed the default behavior

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: storage/storage.go (hunk 1 above)
Comment (hunterlord) [reply storage/storage.go:26]: Sorry, I still don't know how to fix this. Let me close this issue for now.

Comment (hsluoyz) [reply storage/storage.go:26]: @parasiteoflife yes, I like the latter one. So simple and keep compatibility

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**